File: src/main/java/ch/unibas/dmi/dbis/vrem/model/ListExhibitsResponse.java
package ch.unibas.dmi.dbis.vrem.model;

import ch.unibas.dmi.dbis.vrem.model.exhibition.Exhibit;
import java.util.List;

public class ListExhibitsResponse {

    public final List<Exhibit> exhibits;

    public ListExhibitsResponse(List<Exhibit> exhibits) {
        this.exhibits = exhibits;
    }

    public List<Exhibit> getExhibits() {
        return exhibits;
    }
}

File: src/main/kotlin/ch/unibas/dmi/dbis/vrem/model/api/response/ListExhibitsResponse.kt
package ch.unibas.dmi.dbis.vrem.model.api.response

import ch.unibas.dmi.dbis.vrem.model.exhibition.Exhibit
import kotlinx.serialization.Serializable

/**
 * Response to list exhibits action.
 *
 * Wrapper for a list of exhibits.
 *
 * @author loris.sauter
 */
@Serializable
data class ListExhibitsResponse (val exhibits:List<Exhibit>)